signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +414,28 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_daily_transaction_threshold_file_directory() -> PosixPath:
```

Comment:
In this method you return the path with filename. I think it should be renamed (e.g. `_get_daily_transaction_threshold_file`). There is line in `_get_signing_service_daily_transaction_sum_so_far`: `transaction_sum = int(thresholds_directory.read_text()) ` - it suggests you are reading from directory..

BTW. :+1: for general conception of this method. Keeping actual amount in file named as date :)

Reply:
Done